xml/art_solution_automation.xml

```diff
@@ -0,0 +1,438 @@
+        <para>
+          <link xlink:href="https://documentation.suse.com/sbp/all/html/SLES4SAP-hana-sr-guide-PerfOpt-12/index.html"
+            >SAP HANA System Replication Scale-Up - Performance Optimized Scenario (Code 12)</link>
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <link xlink:href="https://www.suse.com/media/white-paper/sap_hana_sr_cost_optimized_scenario_12_sp1.pdf"
+            >Setting up a SAP HANA SR Cost Optimized Infrastructure (SLES-SAP 12 SP1)</link>
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <link xlink:href="https://documentation.suse.com/sbp/all/html/SLES4SAP-hana-sr-guide-PerfOpt-15/index.html"
+            >SAP HANA System Replication Scale-Up - Performance Optimized Scenario (Code 15)</link>
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <link xlink:href="https://documentation.suse.com/sbp/all/html/SLES4SAP-hana-sr-guide-PerfOpt-12_AWS/index.html"
+            >SAP HANA High Availability Cluster for the AWS Cloud </link>
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          <link xlink:href="https://docs.microsoft.com/en-us/azure/virtual-machines/workloads/sap/sap-hana-high-availability"
+            >High availability of SAP HANA on Azure VMs on SUSE Linux Enterprise Server</link>
+        </para>
+      </listitem>
+    </itemizedlist>
+  </sect1>
+
+  <sect1 xml:id="sec-sol-autom-hana-and-netweaver-deploy">
+    <title>&hana; and Netweaver Automated Configuration</title>
+    <para>
+      &salt; formula <package>sapnwbootstrap-formula</package> for bootstrapping
+      and managing the &netweaver; platform.
+    </para>
+    <formalpara>
+      <title>Link:</title>
+      <para>See <link xlink:href="https://github.com/SUSE/sapnwbootstrap-formula"/>.</para>
+    </formalpara>
+    <note>
+      <para>The formula works only with &hana; as database.</para>
+    </note>
+    <itemizedlist>
+      <title>&netweaver; Components</title>
+      <listitem>
+        <para>ASCS instance</para>
+      </listitem>
+      <listitem>
+        <para>ERS instance</para>
+      </listitem>
+      <listitem>
+        <para>PAS instance</para>
+      </listitem>
+      <listitem>
+        <para>AAS instance</para>
+      </listitem>
+      <listitem>
+        <para>Database instance (this adds the required users, tables, views, etc to the current Hana database)</para>
+      </listitem>
+    </itemizedlist>
+    <para>
+      The &salt; formula can be used in these scenarios:
+    </para>
+    <itemizedlist>
+      <title>Scenarios for <package>sapnwbootstrap-formula</package></title>
+      <listitem>
+        <para>Enqueue replication server version 1</para>
+      </listitem>
+      <listitem>
+        <para>Enqueue replication server version 2</para>
+      </listitem>
+      <listitem>
+        <para>Azure, AWS, and GCP automated adjustments for each provider best-practices</para>
```

Comment:
The same than before. This adjustments belong to the HA cluster more than something about NW, but well, maybe they could be specified here too.

Reply:
I'm not entirely sure what you mean. The sentence doesn't contain any HANA string. What part needs to be removed?

Reply:
I don't neither comment anything about HANA hehe
My comment comes because the specific details about the cloud providers are not something that Netweaver really cares. The Netweaver installation and configuration is platform agnostic. 
The cloud adjustments are more about how we configure the HA cluster and its resources, that's why I was wondering if we should put this sentence in the HA cluster formula capabilities.

Reply:
Moved listitem as discussed.